raiden/transfer/channel.py

```diff
@@ -1658,9 +1660,10 @@ def handle_receive_lock_expired_light(
         block_number=block_number,
     )
 
-    events: List[Event] = list()
+    events: List[Event]
```

Comment:
Well the pycharm pep8 checker is indicating that the value of events: List[Event] = [] is useless since it will be always overwritten later, unless you put a condition on the else of course, but this is not the case, so basically it shows a warning saying that the empty list value is never used